homework-classes/ContributorsView.js

```diff
@@ -19,8 +19,46 @@
      * @param {Object[]} contributors An array of contributor objects
      */
     render(contributors) {
-      // TODO: replace this comment and the console.log with your own code
       console.log('ContributorsView', contributors);
+      const contributorsContainer = document.querySelector(
+        '.contributors-container',
+      );
+      const select = document.querySelector('.repo-select');
+      const p = createAndAppend('p', contributorsContainer, {
+        text: 'Contributions',
+      });
+
+      async function createContributorsSection(contributors, parent) {
+        try {
+          contributors.forEach(contribute => {
+            const li = createAndAppend('li', parent);
+            createAndAppend('img', li, {
+              src: contribute.avatar_url,
+              class: 'avatar',
+            });
+            createAndAppend('a', li, {
+              text: contribute.login,
+              href: contribute.html_url,
+              target: '_blank',
+              class: 'cont-name',
+            });
+            createAndAppend('p', li, {
+              text: contribute.contributions,
+            });
+          });
+        } catch (err) {
+          const root = document.getElementById('root');
+          createAndAppend('div', root, {
+            text: err.message,
+            class: 'alert-error',
+          });
+        }
+      }
+      createContributorsSection(contributors, contributorsContainer);
+
+      select.addEventListener('change', () => {
+        contributorsContainer.innerHTML = '';
```

Comment:
You should no reference the `select` element here. It is the exclusive responsibility of the `HeaderView` to deal with that element. Besides, you do not need an event handler simply to erase previous content on each call to the `render` method. Just use this at the top the `render` method:

```js
this.container.innerHTML = '';
```